src/main/java/com/faforever/api/config/error/ErrorResponse.java

```diff
@@ -0,0 +1,15 @@
+package com.faforever.api.config.error;
+
+import lombok.Data;
+
+import java.util.ArrayList;
+
+@Data
+public class ErrorResponse {
+  private ArrayList<ErrorResult> errors = new ArrayList<>();
```

Comment:
Make fields that never change `final` (I feel like I'm repeating myself in every review)